### DISCUSSION TITLE
CI Tests not working

### DISCUSSION THREAD

**Discussion by aditi-v79** [NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11#46] [General]
CI tests are not working, can someone pick it?

**Comment by kaushikjadhav01** [comment]
Created an issue here:
https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/23

**Comment by kaushikjadhav01** [comment]
Issue https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/23 is fixed

**Comment by aditi-v79** [comment]
Tests are fixed locally, but they are still failing on GitHub Actions

**Comment by kaushikjadhav01** [comment]
Created an issue here: https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/47

**Comment by kaushikjadhav01** [comment]
Issue https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/issues/47 fixed in PRs: https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/pull/48 https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/pull/51 https://github.com/NCSU-Fall-2022-SE-Project-Team-11/XpensAuditor---Group-11/pull/52
